domain-server/src/DomainServer.cpp

```diff
@@ -1504,13 +1506,22 @@ QJsonObject jsonForDomainSocketUpdate(const HifiSockAddr& socket) {
     return socketObject;
 }
 
-const QString DOMAIN_UPDATE_AUTOMATIC_NETWORKING_KEY = "automatic_networking";
+void DomainServer::performIPAddressPortUpdate(const HifiSockAddr& newPublicSockAddr) {
+    const QString& publicSocketAddress = newPublicSockAddr.getAddress().toString();
+    const int publicSocketPort = newPublicSockAddr.getPort();
+
+    sendHeartbeatToMetaverse(publicSocketAddress, publicSocketPort);
 
-void DomainServer::performIPAddressUpdate(const HifiSockAddr& newPublicSockAddr) {
-    sendHeartbeatToMetaverse(newPublicSockAddr.getAddress().toString());
+    QString newSettingsJSON = QString("{\"domain_server\": { \"%1\": \"%2\", \"%3\": %4}}")
```

Comment:
Hmm, I think it would be better to do it like this: https://forum.qt.io/topic/65874/create-json-using-qjsondocument/2

I'm not sure if this is actually dangerous, but HifiSockAddr can include a hostname, which is a QString, which might be able to screw up the quoting here. So it'd be better to generate things in such a way that nothing can go wrong with the JSON here.